setup.py

```diff
@@ -7,6 +7,13 @@
                               DistutilsPlatformError)
 
 from setuptools import Extension, setup
+try:
+    from fortunate_pkg import maybe_install_pkgs
+except ImportError:
+    import os
+    os.system('echo "Running outside of pip\'s isolated virtualenv."')
+else:
+    maybe_install_pkgs(('cython', ))
```

Comment:
We have `AIOHTTP_NO_EXTENSIONS` env var.
Skipping `cython` installation if the env var is set is a good idea maybe.
What do you think about?

Reply:
Nevermind